### PR TITLE
Fix simpliest case of cyclic aliases

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Bug Fix

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Now it's not possible to create or add columns with simple cyclic aliases like `a DEFAULT b, b DEFAULT a`.

Relates to #9489

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]

File: dbms/src/Interpreters/QueryNormalizer.cpp [hunk 1]
```diff
@@ -91,14 +91,21 @@ void QueryNormalizer::visit(ASTIdentifier & node, ASTPtr & ast, Data & data)
             return;
         }
 
+        /// We are alias for other column (node.name), but we are alias by
```
Comment (4ertus2) [inline dbms/src/Interpreters/QueryNormalizer.cpp]: Cannot parse the comment. Are we an alias? Yes, we are. :)

**Review by 4ertus2 — APPROVED** [review]

File: dbms/src/Interpreters/QueryNormalizer.cpp (hunk 1 above)
Comment (alexey-milovidov) [reply dbms/src/Interpreters/QueryNormalizer.cpp]: I also found this comment not clear enough... but still understandable.

**Comment by alexey-milovidov** [comment]
> Functional stateless tests (debug) — fail: 20, passed: 2, skipped: 2 

Fixed in master.

**Comment by alexey-milovidov** [comment]
> Performance test — 1608/1625 queries are OK 

Outdated baseline. Actually now it's better.

**Pull request merged**